### DISCUSSION TITLE
pass query sting to app

### DISCUSSION THREAD

**Discussion by redpimpernel** [gravitational/teleport#15796] [Q&A] [answered]
Hi guys, I am trying to understand how Teleport.
I have been trying to setup Webmin behind Teleprt and pass a query string so that I can log in to Webmin without authenticating with Webmin after Teleport.
Query String trying to pass: `"http://localhost:10000/session_login.cgi?user=webminuser&pass=" ` 
(the quotes are in the yaml file)
that is the correct string, as it works with the machine IP address out of Telepot.
the query string just gets striped by Teleport.
Yaml config:
app_service:
  enabled: "yes"`
  apps:
  \- name: "webmin-test"
    uri: "http://localhost:10000/session_login.cgi?user=webminuser&pass=" 
    #public_addr: "webmin-test.teleport.exampledomain.com.au"
    labels:
      org: internal
      env: test
      role: webmin
if anyone has any idea about what I am doing wrong, that would be great. Thank you for your time.

**Comment by webvictim** [comment, marked as answer]
It isn't possible to do this, unfortunately. Teleport will strip the query parameters.
